public/scripts/world-info.js

```diff
@@ -698,7 +962,8 @@ function registerWorldInfoSlashCommands() {
         return '';
     }
 
-    SlashCommandParser.addCommandObject(SlashCommand.fromProps({ name: 'world',
+    SlashCommandParser.addCommandObject(SlashCommand.fromProps({
```

Comment:
My editor hates this kind of formatting, sorry.

Reply:
Mine too. I already prayed Lenny is going to forgive me